common/README.md

```diff
@@ -0,0 +1,69 @@
+# Common Definitions
+
+This folder contains common definitions for protobufs used more than once. We intentionally have this in order to standardize the way we define the data to make it easier for the end user to "plug-n-play".
+
+## Common Proto
+
+TODO
+
+## DEX AMM Proto
+
+TODO
+
+## Token Proto
+
+The token proto is designed to store the token balances of all accounts throughout the blockchains history. It is designed to be network-agnostic and support all tokens. Token's can be blockchain native, erc20s, or any other derivative.
+
+### Definitions
+
+The first part are the token definitions. This is simple, and defines the tokens that are supported in the substream implementation.
+
+> Note: if the token is a native token (e.g. ethereum), then the address is the `0x0` address and the other values need to be hardcoded.
```

Comment:
IIRC, bytes by default serializes without the `0x` prefix in Rust. They are kept this way when we use them in substreams to avoid additional parsing/padding. Any particular reason to store them with `0x`?

Reply:
Will look into this tomo

Reply:
Removed that since it is not something I tackled in this PR